_pages/meetings/fall2020.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: page
+title: 2020 Fall Python in Heliophysics Community Meeting
+permalink: /meetings/fall2020/
+exclude: true
+---
+
+The fall 2020 meeting of the [Python in Heliophysics Community](http://heliopython.org) will be held remotely [**at this Zoom link**](https://cuboulder.zoom.us/j/395871054) (Meeting ID: **395 871 054**). It will be split into seven sessions, spread across four weeks, starting Monday, October 26th, 2020 and ending Monday, November 16th, 2020:
+
+ - **Plenary #1:** Oct 26th 9:00 AM–10:00 AM (MT)
+ - **Tutorials:** Oct 29th 11:00 AM–2:00 PM (MT)
+ - **Plenary #2:** Nov 2nd 9:00 AM–10:00 AM (MT)
+ - **Hackathons:** Nov 4th 7:00 AM–10:00 AM (MT)
+ - **Plenary #3:** Nov 9th 9:00 AM–10:00 AM (MT)
+ - **Unconferences:** Nov 11th 7:00 AM–10:00 AM (MT)
+ - **Plenary #4:** Nov 16th 9:00 AM–10:00 AM (MT)
+
+See [the full agenda](https://docs.google.com/spreadsheets/d/18lrxi8XsjDZ76MjWZPFOovu1JWrdjvoLZmyZfS_LZ3A/edit#gid=0) for exact dates/times of the various sessions. 
+
+### Presentations and Documents
+
+Presentations and documents associated will be kept in the [**meeting folder**](https://drive.google.com/drive/u/0/folders/1T3CGRwXAst8jd7I6xFiKxyCgluGGpg0A) on Google Drive.
```

Comment:
@jibarnum did I miss any links in this section? This line links to the Drive and the three below it link to the only materials I saw.

Reply:
Nope you caught them all I think